xapian-core/include/xapian/types.h

```diff
@@ -30,6 +30,13 @@
 
 namespace Xapian {
 
+/**A unique identifier for a cluster of documents
+ *
+ * clusterid of 0 is invalid, providing an "out of range" value which can
+   be used to mean "not a valid cluster"
+ */
+typedef unsigned XAPIAN_DOCID_BASE_TYPE clusterid;
```

Comment:
Yes you're right. I had added that with a different idea in mind when I started off. Will change that right away.